interview-questions.md

```diff
@@ -14,33 +14,33 @@ Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Global Information Tracker, it's a system that allows you and other people to work on code together and maintain version control so that you don't have a crazy mess of a tracking system. The true value of git is realized the larger an organization gets.
 
-Researched answer:
+Researched answer: same as above but fancier.  answer.toFancierExplanationOfSimpleConcept()
```

Comment:
funny remark. On the next assessment, if you see that your answer is correct, find something interesting that you did not know before.